### ISSUE TITLE
aria-hidden-focus rule should be updated to support HTML inert attribute

### ISSUE THREAD

**Issue by hanu9** [dequelabs/axe-core#3527]
Error - ARIA hidden element must not contain focusable elements

Rule - https://github.com/dequelabs/axe-core/blob/36ace76cd3b3706b19e59dc5168a77a5462e78b6/lib/rules/aria-hidden-focus.json

Product: Axe Extension and axe-core

Expectation: If any element has an inert attribute with aria-hidden this rule should not throw this warning because users can't navigate to the focusable elements. so this error will be false positive in this case.

Actual: Getting error(ARIA hidden element must not contain focusable elements) if we have aria-hidden with inert.

Motivation: all latest browsers support the inert attribute.

**Comment by WilcoFiers** [comment]
Thanks for bringing this up. We are planning to add support for `inert` to axe-core 4.6, which will likely be released around October / November.

My recommendation though, don't rely on it just yet. Use a polyfill. It'll be a while before everyone uses a browser that has it. Firefox doesn't have it yet, and anyone who hasn't updated Chrome / Edge in 3 months doesn't either. It's a really great new feature for accessibility, but until it's widely supported, relying on it can result in some very big accessibility problems.

Closing this, as we have another issue for it here: #3448
